packages/runtime/test/fixtures/escrow-account/assets/crowdFundApproval.teal

```diff
@@ -0,0 +1,470 @@
+#pragma version 4
```

Comment:
This teal file looks familiar, Where did you get these code?

Reply:
It was in some other asset folder. I couldn't access it directly from there bcz of fixtures. So, I added this file over here.